text/0000-icons.md

```diff
@@ -93,10 +120,6 @@ The optimized SVG icons will be made available using the `files` attribute of pa
 
 Implementing for each framework means supporting multiple formats, increasing up-front and maintenance effort for each additional framework and potentially less flexibility/visibility from the client side.
```

Comment:
I don't think this should be a drawback. If we design this project correctly, we should just be adding new "configurations" vs having less flexibility.

Reply:
I'm just saying that it adds a level of overhead for each framework - each one requires knowledge and effort to be built and maintained to the specifics of that framework.  it may not be *much* overhead, but more than if we provided a single solution that could be used by any framework (though most of those solutions just push the effort off to the clients, and we don't want to do that).

That may possibly be entirely handled by transpilers and configuration, but I couldn't say for certain.

Reply:
Should I remove this?

Reply:
@Rykus0 I would because it should be the job of a script, not us. 😉